styles.css

```diff
@@ -1 +1,84 @@
-/*Maak hier jouw CSS!*/
\ No newline at end of file
+/*Maak hier jouw CSS!*/
+
+
+/*styling*/
+body {background-color: #F0E9E4}
+
+/*styling*/
+section {
+    border-style: solid;
+    border-width: 2px;
+    border-radius: 5px;
+    border-color: #304D6D;
+    background-color: #82A0BC;
+    padding: 2px;
+}
+
+/*grid start*/
+div {
+    grid-gap: 16px;
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows:
+        /*first try:
+        50px 190px 100px 50px*/
```

Comment:
Ik denk dat je eerste poging beter was, aangezien ze in de opdracht aangeven dat de eerste en twee rij in totaal 240px moeten zijn (dus inderdaad 50px + 190px). Zoals je hem nu hebt geschreven wordt de uiteindelijke height van vak C kleiner dan 240px